eclipse/plugins/org.switchyard.tools.ui.editor/src/org/switchyard/tools/ui/editor/components/camel/xml/NewCamelXMLRouteCompenentWizard.java

```diff
@@ -92,13 +108,70 @@ String getSpringStubText(String serviceName) {
         sb.append("</beans>\n");
         return sb.toString();
     }
-    
+
+    private InputStream getOldRouteStubText() {
+        ByteArrayOutputStream baos = null;
+        try {
+            DocumentRoot doc = SpringFactory.eINSTANCE.createDocumentRoot();
+            RoutesDefinition routes = SpringFactory.eINSTANCE.createRoutesDefinition();
+            RouteDefinition route = SpringFactory.eINSTANCE.createRouteDefinition();
+            route.setId("_route1");
+            FromDefinition from = SpringFactory.eINSTANCE.createFromDefinition();
+            from.setId("_from1");
+            LogDefinition log = SpringFactory.eINSTANCE.createLogDefinition();
+            log.setId("_log1");
+            String serviceName = getService().getName();
+
+            from.setUri("switchyard://" + serviceName); //$NON-NLS-1$
+            log.setMessage("" + serviceName + " - message received: ${body}"); //$NON-NLS-1$ //$NON-NLS-2$
+
+            doc.setRoutes(routes);
+            doc.setRoute(route);
+            route.getFrom().add(from);
+            route.getLog().add(log);
+            routes.getRoute().add(route);
+
+            doc.getXMLNSPrefixMap().put("", SpringPackage.eNS_URI); //$NON-NLS-1$
+
+            Resource routeResource = new SpringResourceFactoryImpl().createResource(URI.createGenericURI("temp", //$NON-NLS-1$
+                    "temp", null)); //$NON-NLS-1$
+            routeResource.getContents().add(doc);
+
+            baos = new ByteArrayOutputStream();
```

Comment:
Unfortunately we are stuck at a 1.6 compliance level for right now. We will address this moving forward in the next release. But it's a good thought and we'll definitely utilize the try-resource approach.